src/isPalindrome.js

```diff
@@ -19,7 +19,17 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  const lettersArray = str.match(/\w/gi);
+  const IsNotEmptyArray = lettersArray !== null;
```

Comment:
1. IsNotEmptyArray -- you dont' use this variable so it's not really useful.
2. variable names are lowerCamelCase. which means: isNotEmptyArray.